lib/CMakeLists.txt

```diff
@@ -22,7 +22,10 @@ set(Python_FIND_IMPLEMENTATIONS CPython PyPy)
 find_package(Benchmark)
 find_package(Boost 1.67 REQUIRED)
 find_package(Eigen3 REQUIRED)
-find_package(Sanitizers REQUIRED)
+option(SANITIZERS "Enable sanitizers" OFF)
```

Comment:
Should we change the regular CI and/or dev setups to enable this by default? For example in the cmake presets?